storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -192,12 +195,23 @@ func createAttributes(resource pcommon.Resource, attributes pcommon.Map, scope s
 
 	if settings.EnableTypeAndUnitLabels {
 		unitNamer := otlptranslator.UnitNamer{UTF8Allowed: settings.AllowUTF8}
+
 		if metadata.Type != prompb.MetricMetadata_UNKNOWN {
-			l["__type__"] = strings.ToLower(metadata.Type.String())
+			typeValue := strings.ToLower(metadata.Type.String())
```

Comment:
Not sure if I'm missing something, but `typeValue` isn't used anywhere... Could we revert this change, then?